.github/workflows/tests.yml

```diff
@@ -0,0 +1,31 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  testsuite:
+    name: all tests
+    runs-on: ubuntu-latest
+    strategy:
+      # This prevents cancellation of matrix job runs, if one or more already failed
+      # and let the remaining matrix jobs be executed anyway.
+      fail-fast: false
+    env:
+      php: '8.1'
```

Comment:
Why is this choosen to be set as "env" instead of providing it as a matrix flavour ? Would make it easier in the feature to support two php versions.

Just asking. However, would use "phpversion" instead to avoid confusing/clashes, which may occur.

Reply:
We do not need a PHP matrix as all we do is some linting and we lint for the bottom most PHP Version that is supported?

Reply:
ok.